.github/workflows/CI.yml

```diff
@@ -11,12 +11,6 @@ jobs:
       matrix:
         platform: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [14]
-        include:
-          - platform: ubuntu-latest
-            node-version: 12
-          - platform: ubuntu-latest
-            node-version: 10
-    runs-on: ${{ matrix.platform }}
```

Comment:
we still need this line to test cross platform

Reply:
Yeah, caught that! :)